### DISCUSSION TITLE
Affix prefix components inside inputgroup

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/chakra-ui#8204] [Q&A]
Hi guys, how I can adjust this to have prefix a select and affix a button? I use Inputgroup:

![Screenshot 2024-01-10 alle 12 15 25](https://github.com/chakra-ui/chakra-ui/assets/40239902/d8789290-0c89-402a-b2cb-a929f69a9592)

**Comment by segunadebayo** [comment]
Use `InputGroup` with `startElement` and `endElement`.
